pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>org.folio</groupId>
   <artifactId>folio-kafka-wrapper</artifactId>
-  <version>2.7.0-SNAPSHOT</version>
+  <version>2.7.1-SNAPSHOT</version>
```

Comment:
If we need to release it as a bugfix, let's just merge it to the release branch as well as to the master, but not change the master version

Reply:
rolled back the version